### PR TITLE
Support Kubernetes v1.11

### PR DESCRIPTION
**Pull request by andrewsykim**
Is still backward compatible with Kubernetes v1.9 & v1.10

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -27,11 +27,15 @@
 
 [[override]]
   name = "k8s.io/apiextensions-apiserver"
-  version = "kubernetes-1.10.2"
+  version = "kubernetes-1.11.0-rc.3"
```
[the remark this answers is not in the document]
Comment (andrewsykim) [reply Gopkg.toml]: Thanks for the catch, one of the dependencies was late on the release so I was using rc3 temporarily. Fixed now.

**Comment by lxfontes** [comment]
🎉

**Comment by bhcleek** [comment]
did you `dep prune`? If so, then lgtm

**Comment by andrewsykim** [comment]
I did dep prune :)

**Pull request merged**